package.json

```diff
@@ -17,15 +18,21 @@
   },
   "homepage": "https://github.com/maxwellito/breaklock",
   "devDependencies": {
-    "babel-core": "^6.24.0",
-    "babel-loader": "^6.4.1",
-    "babel-preset-es2015": "^6.24.0",
+    "@babel/core": "^7.11.1",
+    "@babel/preset-env": "^7.11.0",
+    "babel-loader": "^8.1.0",
     "css-loader": "^0.28.0",
-    "extract-text-webpack-plugin": "^2.1.0",
+    "extract-text-webpack-plugin": "^3.0.2",
```

Comment:
Is it still in use? If not, happy to ditch it :)

Reply:
```suggestion
```